Documentation/Input/Pointers.md

```diff
@@ -78,9 +78,9 @@ The MousePointer powers a screen to world raycast for far interactions, but for
 
 #### Near pointers
 
-##### [`PokePointer`](xref:Microsoft.MixedReality.Toolkit.Input.PokePointer)
+##### PokePointer
```

Comment:
Why remove the xref? Also why remove the xref only on the near pointers header titles?

Reply:
Couple reasons:

1) When the header was marked with the monospace annotation, it was not clear that I could click it
2) The monospace styling made me not realize it was a header, visually I wasn't able to break up the text into sections

I ended up removing the link for consistency -- other headers didn't have links.

Reply:
hmm, I guess I assume anything with the ticks is a clickable xref. Whichever path is selected, all docs should follow it.

For example, it's weird in this page the other points headers are links but not near pointers. Whatever is the usage, it should be consistent

Reply:
Okay, reverted the change to stay consistent

Reply:
Made all links to xrefs have ticks.